src/main/features/ccj/form/models/qualifiedDeclaration.ts

```diff
@@ -0,0 +1,56 @@
+import { IsDefined, MaxLength } from 'class-validator'
+import { IsBooleanTrue } from 'forms/validation/validators/isBooleanTrue'
+import { SignatureType } from 'app/common/signatureType'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import * as toBoolean from 'to-boolean'
+
+/**
+ * We cannot reuse StatementOfTruth class as for legal reason error message must be different.
+ */
+export class ValidationErrors {
+  static readonly DECLARATION_REQUIRED: string = 'Please select I confirm that I believe the details I have provided are correct.'
+  static readonly SIGNER_NAME_REQUIRED: string = 'Enter the name of the person signing the declaration'
+  static readonly SIGNER_NAME_TOO_LONG: string = 'You’ve entered too many characters'
+  static readonly SIGNER_ROLE_REQUIRED: string = 'Enter the role of the person signing the declaration'
+  static readonly SIGNER_ROLE_TOO_LONG: string = 'You’ve entered too many characters'
+}
+
+export class QualifiedDeclaration {
+  type: string = SignatureType.QUALIFIED
```

Comment:
Maybe `QualifiedDeclaration` should extend declaration and then we don't have to duplicate this field?

Reply:
we would still have to set the field though?

Reply:
done